subscriptionform/static/css/subscription.css

```diff
@@ -0,0 +1,13 @@
+h2 {
+    color: #5f5d8a;
+}
+
+body {
+    padding-left: 15px;
+    background-color: #c6e7ff;
+}
+
+p {
+    font-family: allerta, Arial, "Helvetica Neue", Helvetica, "sans-serif";
```

Comment:
Ja! Als er whitespace in de naam van het font zit voor de zekerheid even quotjes omheen, anders hoeft hiet niet. Mág wel, kwestie van persoonlijke smaak :)